### ISSUE TITLE
Assorted description errors

### ISSUE THREAD

**Issue by UndeadDevel** [LoneGazebo/Community-Patch-DLL#5120]
_**1. Mod version** (i.e Date - 4/23):_
11-30-18

_**2. Mod list** (if using Vox Populi only, leave blank):_

VP (EUI)
City-States Leader for VP (v8)
List Enhancements for VP (v4.2)
Trade Opportunities for VP (v8.1)
Reforestation (v8) (slightly modified by me)
Civ-linked GG/GA (v11)
Enhanced Naval Warfare for VP (v1.6)
Enlightenment Era (VP) (v1.2)
Faster Aircraft Animations (v3)
Capture Great People (I neutered this one because of compatibility issues)
Espionage Mapping (v7) (modified by me)
Religious Settlers (v1)
Improved Air Recon (v1)
Missiles No Supply (v1)
More Unique Components for VP (v38)
Promotions Flags EUI (v1.1) (slightly modified by me)
Subs Ignore Borders (v3)
Subs Immune Under Ice (v2)
Destination (v4)
Filters in Trade Route Overview (VP) (v1.3) (slightly modified by me)
Improved City View (VP EUI) (v11.0)
Promotion Tree (v17)
Upgrade Tree (v9)
VP Promo Icons (v1.9)

_**3. Error description**:_
Some more description errors I found (note: I'm playing on a Large map and Epic speed, in case that matters):
- the "Scavenger" promotion states that pillaging a tile gives 5 XP, while it actually gives 7 XP
- the "Spotters" promotion states that staying in enemy territory gives 2 XP, while it actually gives 3 XP
- the description for the strategic monopoly on Oil states that "Units gain +5 XP after combat.", when it's actually +2 XP

Supporting information:
Please note that you can attach .zip files by dragging-and-dropping them. If possible, zip up all supporting data and post that way.

1. Log files (always attach your Logs folder, located at My Documents/My Games/Sid Meier's Civilization 5. Make sure you have enabled logging before experiencing an error! Go here to find out how: http://forums.civfanatics.com/showthread.php?t=487482):

2. Save game (always attach a save that was made a turn before the error; located at My Documents/My Games/Sid Meier's Civilization 5/ModdedSaves):

3. CvMiniDump.dmp file (attach if experiencing a game crash. Located at Program Files/Steam/steamapps/common/Sid Meier's Civilization V):

4. Screenshots (optional):

**Comment by LoneGazebo** [comment]
Values look fine on my end - probably a mod conflict.

**Comment by UndeadDevel** [comment]
Hmm...by fine do you mean the values that I described as "actually happening" are set in your version or the other ones?

**Comment by UndeadDevel** [comment]
Okay I checked this in the code here; it's not a mod conflict issue.

The strategic monopoly text for Oil is indeed fixed (was fixed in December by you, so my VP is simply too old); the other two, however, are not:

TXT_KEY_PROMOTION_SCOUT_XP_PILLAGE_HELP 
and 
TXT_KEY_PROMOTION_SCOUT_XP_SPOTTING_HELP

in file [UIText.xml](https://github.com/LoneGazebo/Community-Patch-DLL/blob/72137235dbab0c78d0c65a4b2ea33bad85b9ef61/Community%20Balance%20Patch/Balance%20Changes/Text/en_US/UIText.xml) are not correct.

Or is it that scouts are actually supposed to get 2 XP per turn from Spotters and 5 XP from Pillage?

**Comment by LoneGazebo** [comment]
It's a mapspeed thing, all XP is scaled.

**Comment by UndeadDevel** [comment]
Right, I half suspected this, because it was 1.5 times as much, which fits the game speed multiplier.

Is it possible to make the ingame descriptions reflect that, however, so players don't get confused?
And why doesn't the Oil monopoly thing scale?
